guide/ops/locations/_openstack.md

```diff
@@ -1,18 +1,66 @@
 ---
-section: Openstack
-title: Openstack
+section: OpenStack
+title: OpenStack
 section_type: inline
 section_position: 6
 ---
 
-## Openstack
+## OpenStack
 
 ### Apache jclouds
 
 Support for OpenStack is provided by Apache jclouds. For more information, see their guide
 [here](https://jclouds.apache.org/guides/openstack/).
 
 
+### Connection Details
+
+The endpoint URI is that of keystone (normally on port 5000).
+
+The identity normally consists of the tenant and username, colons-separated. The credential is 
+the password. For example:
+
+    location:
+      jclouds:openstack-nova:
+        endpoint: http://x.x.x.x:5000/v2.0/
+        identity: "your-tenant:your-username"
+        credential: your-password
+
+OpenStack Nova access information can be downloaded from the openstack web interface, for example 
+as an openrc.sh file. This file will normally contain the identity and credential.
+
+
+### Common Configuration Options
+
+Below are examples of configuration options that use values specific to OpenStack environments:
+
+* The `imageId` is the id of an image. For example,
+  `imageId: RegionOne/08086159-8b0b-4970-b332-a7a929ee601f`.
+  These ids can be found from the the CLI or the web-console, for example in IBM Blue Box, 
```

Comment:
I think this link is good enough. I don't think there is a good "public" OpenStack url to use for this (e.g. the rackspace user interface is completely different from the normal OpenStack UI). The advantage of the IBM Blue Box link is that it is a half-way house: it's enough like what a private OpenStack enterprise would have, but is also a public URL.